README.md

```diff
@@ -323,6 +323,7 @@ CSS Beautifier Options:
   -e, --eol                          Character(s) to use as line terminators. (default newline - "\\n")
   -n, --end-with-newline             End output with newline
   -L, --selector-separator-newline   Add a newline between multiple selectors
+  -C, --curly-start-newline          Add a newline before the opening { bracket of selector(s)
```

Comment:
What do you think of reusing the `brace-style` setting instead of create a new field specific to the css beautifier? 
I ask because the formatting `curly-start-newline` creates is the same as `brace-style=expand`.  The default behavior matches `brace-style=collapse`  (or `end-expand` but these are the same in css.) 

Don't change anything yet, I'd like to discuss and see if what I'm thinking makes sense to you.

Reply:
Thanks for looking at this!

Think it would be more consistent to use `brace-style` as well (I missed it out before).
Perhaps a trimmed set of options like `[collapse|expand|none]`, since as you mentioned `end-expand` dosen't make sense in css. I'm not too sure if `preserve-inline` makes sense though. (adding css to a minified file while preserving the minified styles maybe?)

I could expand the scope of this to include support for these if needed, or in separate PRs. Let me know what you think!

Reply:
@ang-zeyu
I think it would be fine to start with support for just `collapse` and `expand` (and treat `end-expand` and `none` the same as `collapse`).  Then you could open an issue to add support for `preserve-inline` and `none` and someone (or you) can do that at a later date in another PR. 

That will get this cool new feature out for people to use with minimal added work for you now.  Sound good?

Reply:
Yup! will update this with `brace-style=[collapse|expand]` as a start then